Source/WebKit/NetworkProcess/cocoa/NetworkTaskCocoa.mm

```diff
@@ -142,18 +142,18 @@ static bool shouldCapCookieExpiryForThirdPartyIPAddress(const WebCore::IPAddress
 }
 
 #if HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)
-void NetworkTaskCocoa::setCookieTransformForFirstPartyRequest(const WebCore::ResourceRequest& request)
+void NetworkTaskCocoa::setCookieTransformForThirdPartyRequest(const WebCore::ResourceRequest& request, IsRedirect isRedirect)
 {
     if (!request.isThirdParty())
```

Comment:
It seems like this should be an assertion, and it seems like NetworkTaskCocoa::setCookieTransformForFirstPartyRequest should have a complimentary assertion.